src/main/java/jpass/data/DocumentHelper.java

```diff
@@ -111,7 +111,7 @@ public Entries readDocument() throws FileNotFoundException, IOException, Documen
             if (this.key == null) {
                 inputStream = new FileInputStream(this.fileName);
             } else {
-                inputStream = new GZIPInputStream(new CryptInputStream(new FileInputStream(this.fileName), this.key));
+                inputStream = new CryptInputStream(new FileInputStream(this.fileName), this.key);
```

Comment:
Instead of removing the GZIPInputStream you could wrap the FileInputStream e.g. into a SaltInputStream:
`new GZIPInputStream(new CryptInputStream(new SaltInputStream(new FileInputStream(this.fileName)), this.key));`